guidance/include/guidance/guidance_state_machine.hpp

```diff
@@ -71,15 +71,57 @@ namespace guidance
              */
             uint8_t getCurrentState();
 
+            virtual void StartUpState(Signal signal) = 0;
+            virtual void DriversReadyState(Signal signal) = 0;
+            virtual void ActiveState(Signal signal) = 0;
+            virtual void EngagedState(Signal signal) = 0;
+            virtual void InactiveState(Signal signal) = 0;
+            virtual void OffState(Signal signal) = 0;
+            // a local variable keeps the current state machine state
+            State currentGuidanceState;
```

Comment:
According to the ROS Style guide member variables should be underscored and have trailing underscores. So this would be current_guidance_state_

Reply:
currentGuidanceState is legacy, I think then its needed to be update though out all the files.

Reply:
That's fine you can go ahead and update it.